src/planscape/forsys/io.py

```diff
@@ -0,0 +1,61 @@
+from django.contrib.gis.geos import Polygon
+from django.http import JsonResponse, QueryDict
+
+import json
+
+
+# Request parameters for a Forsys scenario set call.
+# The following URL parameters are parsed:
+# save_debug_info
+#   Whether or not debug information should be returned in the HTTPResponse
+#   The value can be 0, 1, True, or False.
+# region
+#   The planning region.
+#   The value is one of the following strings:
+#       TODO: list the regions
+# priorities
+#   Conditions to be prioritized when estimating benefit scores.
+#   The value is a comma-separated list consisting of a subset of the following conditions:
+#       TODO: list the conditions
+class ForsysScenarioSetRequestParams():
+    def _get_default_polygons() -> list[Polygon]:
+        p1 = Polygon(((-120.14015536869722, 39.05413814388948),
+                     (-120.18409937110482, 39.48622140686506),
+                     (-119.93422142411087, 39.48622140686506),
+                     (-119.93422142411087, 39.05413814388948),
+                     (-120.14015536869722, 39.05413814388948)))
+        p1.srid = 4269
+        p2 = Polygon(((-120.14015536869722, 38.05413814388948),
+                     (-120.18409937110482, 38.48622140686506),
+                     (-119.93422142411087, 38.48622140686506),
+                     (-119.93422142411087, 38.05413814388948),
+                     (-120.14015536869722, 38.05413814388948)))
+        p2.srid = 4269
+        return [p1, p2]
+
+    def __init__(self, params: QueryDict) -> None:
+        if params.has_key('save_debug_info'):
+            self.save_debug_info = bool(params['save_debug_info'])
+        if params.has_key('region'):
+            self.region = params['region']
+        if params.has_key('priorities'):
+            self.priorities = self.priorities = ','.split(params['priorities'])
+
+    # ------------------------------------------
+    # Parameters (and their default assignments)
+    # ------------------------------------------
+    # Whether or not debug information should be returned in the HTTPResponse.
+    # url parameter is 'save_debug_info', value can be 0, 1, True, or False.
+    save_debug_info: bool = False
+    # The planning region.
+    # url parameter is 'region', value is a string.
+    region: str = 'sierra_cascade_inyo'
+    # Conditions to be prioritized when estimating benefit scores.
+    # url parameter is 'priorities', value is a comma-separated list of conditions.
+    priorities: list[str] = ['fire_dynamics',
+                             'forest_resilience', 'species_diversity']
+    # The project areas to be compared.
+    # url parameter is 'project_areas', value is a json whose first layer is {'}
+    project_areas: list[Polygon] = [__get_default_polygon()]
```

Comment:
@strangeties do you want me to pass you a project area json or would you prefer to read it from the DB?

Reply:
I hope this class can support both - json for debugging, db for production: wdyt?

Reply:
SG, let's include both